test/e2e/suplies.spec.ts

```diff
@@ -1,9 +1,9 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { INestApplication } from '@nestjs/common';
 import * as request from 'supertest';
-import { AppModule } from './../src/app.module';
+import { AppModule } from './../../src/app.module';
 
-describe('AppController (e2e)', () => {
+describe('Supplies (e2e)', () => {
```

Comment:
Poderias remover esse teste? Ele vai falhar a depender da configuração do banco.

Uma outra alternativa era criar um ' health check' endpoint . Assim continuariamos tendo um exemplo de teste 'e2e' funcionando.

Reply:
Sim, esse endpoint está ai só para manter o exemplo para os testes e2e para quem for usar.
Não gostaria de remover enquanto não temos outros testes nessa área (para manter o exemplo).

O correto na realidade seria o teste não falhar por conta do DB.
Não sei ao certo como é essa parte com o Prisma (nunca usei ele), mas acredito que o ideal seria a suite de testes inicializar um DB limpo e "resetar" as tabelas a cada teste para ter algo mais previsível.

Reply:
Ok, daqui a pouco faço a alteração.

Reply:
@AlbuquerqueRafael 
Como pode ver, mantive o teste mas removi a checagem da resposta para evitar o erro (com um aviso de `@todo` para fazer essa parte futuramente, assim é feita apenas a verificação da resposta 200.